products/views.py

```diff
@@ -42,4 +44,58 @@ def get(self,request):
 
         } for product in products]
 
-        return JsonResponse({'results':results}, status=200)
\ No newline at end of file
+        return JsonResponse({'results':results}, status=200)
+
+class ProductDetailView(View):
+    def get(self, request, product_id):
+        product                 = Product.objects.get(id=product_id)
+        product_info            = product.productinformation
+        allergy_info            = AllergyProduct.objects.filter(
+            product_information=product_info.id
+        )
+        
+        allergy_list            = [{
+            'id'      : '{}'.format(index+1),
+            'allergy' : allergy_info[index].allergy.name
+        } for index in range(len(allergy_info))]
```

Comment:
```suggestion        
        allergy_list            = [{
            'id'      : allergy.id,
            'allergy' : allergy.name
        } for allergy in product_info.allergy.all()]
```
중간테이블을 따로 색인할게 아니라. MtoM 적용했다면 그냥 이렇게 하는게 맞지 않나요?

Reply:
수정 반영하겠습니다.   MtoM으로 모델링을 했음에도 불구하고, 실제 데이터를 가져올 때 활용하질 못했네요...